### PR TITLE
Add an ENV to configure initial context

### PR DESCRIPTION
**Pull request by fabdouglas**
Add an ENV to configure initial context and avoid a manual step either with a "docker exec" either with a temporary UI access with root (/) context to change it.
Default value of NEXUS_CONTEXT is "/".

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,18 +28,22 @@ RUN mkdir -p /opt/sonatype/nexus \
...
     -e "s|karaf.home=.|karaf.home=/opt/sonatype/nexus|g" \
     -e "s|karaf.base=.|karaf.base=/opt/sonatype/nexus|g" \
     -e "s|karaf.etc=etc|karaf.etc=/opt/sonatype/nexus/etc|g" \
     -e "s|java.util.logging.config.file=etc|java.util.logging.config.file=/opt/sonatype/nexus/etc|g" \
     -e "s|karaf.data=data|karaf.data=${NEXUS_DATA}|g" \
     -e "s|java.io.tmpdir=data/tmp|java.io.tmpdir=${NEXUS_DATA}/tmp|g" \
-    -i /opt/sonatype/nexus/bin/nexus.vmoptions
+    -i /opt/sonatype/nexus/bin/nexus.vmoptions \
+  && sed \
+    -e "s|nexus-context-path=/|nexus-context-path=/nexus|g" \
```
Comment (dbradicich) [inline Dockerfile]: i believe this should be ```-e "s|nexus-context-path=/|nexus-context-path=${NEXUS_CONTEXT}|g" \```

File: Dockerfile [hunk 2]
```diff
@@ -28,18 +28,22 @@ RUN mkdir -p /opt/sonatype/nexus \
   && curl --fail --silent --location --retry 3 \
     https://download.sonatype.com/nexus/3/nexus-${NEXUS_VERSION}-unix.tar.gz \
   | gunzip \
-  | tar x -C /opt/sonatype/nexus --strip-components=1 nexus-${NEXUS_VERSION} \
+  | tar -x -C /opt/sonatype/nexus --strip-components=1 nexus-${NEXUS_VERSION} \
```
Comment (dbradicich) [inline Dockerfile]: this change seems unrelated and unrequired (```x``` or ```-x``` are valid)

**Comment by dbradicich** [comment]
+1 to the PR (after couple comments resolved above) thanks!

**Comment by RobWin** [comment]
Any progress on this PR?

**Comment by DarthHater** [comment]
@fabdouglas hi there!

Firstly, thanks for the contributions. Secondly, can you fill out a CLA, sign and send it to me: jhesse at sonatype dot com

http://www.sonatype.org/SonatypeCLA.pdf

Once we have that I'll try and get this merged down!

**Comment by DarthHater** [comment]
Closing in favor of #40
